config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -2,6 +2,7 @@ presubmits:
   kubernetes-sigs/cloud-provider-azure:
   - name: pull-cloud-provider-azure-check
     always_run: true
+    skip_if_only_changed: "^docs/|^site/|^\\.github/|\\.(md|adoc)$|^(README|LICENSE)$"
```

Comment:
will this work? I would also like to configure this for the other upstream repos if it works.

Reply:
Please refer to https://github.com/kubernetes/test-infra/blob/c341b223083a7e5766be99620eba58cc3c8142f1/prow/jobs.md#triggering-jobs-based-on-changes
I have no idea how to test it.

Reply:
sure so let's merge it first

Reply:
I think it works. Please refer to https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1139